specification/azurestackhci/resource-manager/readme.md

```diff
@@ -85,6 +92,25 @@ input-file:
   - Microsoft.AzureStackHCI/preview/2023-09-01-preview/virtualHardDisks.json
   - Microsoft.AzureStackHCI/preview/2023-09-01-preview/virtualMachineInstances.json
 ```
+### Tag: package-preview-2023-08
+
+These settings apply only when `--tag=package-preview-2023-08` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-08'
+input-file:
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/arcSettings.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/clusters.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/extensions.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/offers.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/operations.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/publishers.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/skus.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/updateRuns.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/updateSummaries.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/updates.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/deploymentSettings.json
+  - Microsoft.AzureStackHCI/preview/2023-08-01-preview/edgeDevices.json
```

Comment:
@priyjain358  Is your change for this PR  just adding arcSettings, clusters, extensions back and add new resources ? If yes, could you support the 2023-09-01-preview version and add it to the package-preview-2023-09, we've already released SDK with that tag, and are not allowed to release an older version.

Reply:
With that, we could just release a minor version to add those back.

Reply:
@qiaozha, we didn't raise any SDk release request, when did we release SDK with 2023-09-01-preview version? It's still in private preview..

Reply:
We have already released 2023-08-01-preview in private and want to release it as public preview now, hence we can't include those changes in 2023-09-01-preview. 
If we don't change the default tag, what are the implications? Will we be able to release SDK with 08-01-preview?